src/packages.jl

```diff
@@ -453,6 +453,69 @@ function build_recursive(name::String; quiet::Bool = false,
   return true
 end
 
+"""
+    test(name::String)
+
+Return a boolean indicating if the GAP package with name `name` succeeds
+in running its tests.
```

Comment:
I think we should not insist on using the `Return ...` format here, as the result is IMHO somewhat unnatural and makes it unclear whether the tests are actually run, which arguably is the main point of this function. How about this:
```suggestion
Run the tests of GAP package `name` and return a boolean indicating whether
they succeeded (`true`) or not.
```